src/main/java/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer.java

```diff
@@ -28,31 +32,40 @@ public class StaticAnalyzer extends Scanner {
 	private static final String STATIC_ANALYZER = "Static Analyzer"; //$NON-NLS-1$
         
         private boolean m_openSourceOnly;
+        private String m_includeSCAGenerateIRX;
+        private boolean m_additionalOptionsUploadDirect;
+        private boolean m_includeSCAUploadDirect;
```

Comment:
I think we can simplify this with shorter variable names and we only need 2 new member variables:
```
private boolean m_includeSCA;
private boolean m_uploadDirect;
```
The value of m_includeSCAGenerateIRX is just `m_includeSCA && !m_uploadDirect`. Likewise, the value of m_includeSCAUploadDirect is just `m_includeSCA && m_uploadDirect`. So I don't think we need a third variable here.

Reply:
Matt, Are you referring that we should use the same name for both the checkbox(under "generateIRX" & "upload Files & folders" scan menthod) in config file? 
1. There is difference in the UI names for the both the checkboxes.
2. Third variable is for the value of the optionalBlock checkbox under the "upload files & folders" scan method. We can remove that optionalBlock as it contains only 1 element into it.

Reply:
I have changed the variable name for the additional option under "upload Files & folders" scan method. New variable name would be "m_hasOptionsUploadDirect".
Also, changed the variable type of the "m_includeSCAUploadDirect" to String so that we have a parity between the both checkboxes of "includeSCA".